### DISCUSSION TITLE
是不是不支持 vless+reality ?

### DISCUSSION THREAD

**Discussion by xhshan** [v2rayA/v2rayA#1552] [General]
是不是不支持 vless+reality ?  同样的配置我放到v2rayN里面就可以，放v2rayA里面测试的时候就是NOT STABLE

**Comment by chendechang97** [comment]
俺也一样，ping是正常的 http就失败了

**Reply by lin1943** [reply]
docker好像不行？

**Comment by SaveMyBlackberries** [comment]
Same here...

**Comment by 189er** [comment]
一元机场.vip   这里全是 vless+reality  ,我用着很好啊

**Reply by pooneyy** [reply]
+1，我也是。使用docker部署，使用的默认配置，一元机场.vip 和 小小布吉岛 的 vless(tcp+reality) 都能连上

**Comment by SaveMyBlackberries** [comment]
我已经换成sing-box了速度很快很稳定

**Reply by xyli09** [reply]
这个不支持singbox吧

**Reply by SaveMyBlackberries** [reply]
sing-box支持vless-reality的

**Reply by lovefan-fan** [reply]
sing-box有类似这个的项目吗

**Comment by testonlys** [comment]
确实不支持

**Comment by lhz07** [comment]
reality只有xray核心才支持，参考[https://v2raya.org/docs/manual/variable-argument/](url)更换xray核心

**Reply by pooneyy** [reply]
我使用docker部署，就用的这一页 https://v2raya.org/docs/prologue/installation/docker/ 给的配置， vless(tcp+reality) 能正常连上

**Reply by lin1943** [reply]
正解，更换到xray可用

**Reply by Menlonpi** [reply]
谢谢，更换到xray可用

**Reply by xdzgithub** [reply]
楼上那个官网链接是对的，但需要docker启动时，把
  -e V2RAYA_V2RAY_BIN=/usr/local/bin/v2ray \
改成
  -e V2RAYA_V2RAY_BIN=/usr/local/bin/xray \
官网写了默认是xray，但实际没有改

**Comment by woeoio** [comment]
确实不支持 vless 协议

**Comment by maomao1460** [comment]
换成xray了，还是不行

**Reply by lin1943** [reply]
我上次更新后不能用，现在用的2.2.6.3，你试试

**Reply by maomao1460** [reply]
还是不行，一样的报错，哎

**Reply by hainingning** [reply]
实测安装xray后什么配置都不用改，自动就可以用了

**Reply by wojiaxiaojinyu** [reply]
> 实测安装xray后什么配置都不用改，自动就可以用了

如何更换xray内核

**Comment by laopaoer-wallet** [comment]
vless(tcp+reality)  不支持。

**Reply by hainingning** [reply]
使用xray内核可以使用
[v2raya wiki](https://v2raya.org/docs/manual/variable-argument/#systemd-%E7%AE%A1%E7%90%86%E7%9A%84-v2raya)

**Comment by zhangjunjie20022002-cyber** [comment]
换xray可以，但是xray不支持负载均衡了
